cmdutil/https/config.go

```diff
@@ -1,13 +1,17 @@
 package https
 
+import "time"
+
 // Config for HTTP and HTTPS servers.
 type Config struct {
 	InsecurePort int `env:"HEROKU_ROUTER_HTTP_PORT,required"`
 	SecurePort   int `env:"HEROKU_ROUTER_HTTPS_PORT,required"`
 
 	// These environement variables are automatically set by ACM in
 	// relation to Let's Encrypt certificates.
-	ServerCert  string `env:"SERVER_CERT,required"`
-	ServerKey   string `env:"SERVER_KEY,required"`
-	UseAutocert bool   `env:"HTTPS_USE_AUTOCERT"`
+	ServerCert   string        `env:"SERVER_CERT,required"`
+	ServerKey    string        `env:"SERVER_KEY,required"`
+	UseAutocert  bool          `env:"HTTPS_USE_AUTOCERT"`
+	ReadTimeout  time.Duration `env:"READ_TIMEOUT"`
+	WriteTimeout time.Duration `env:"WRITE_TIMEOUT"`
```

Comment:
Are there defaults we could apply here that represent best practices or upper bounds? For example, something that matches how we configure ELBs or the timeouts we use in the dogwood router?

Definitely not a blocker, making it straightforward to configure is a great start (and less risky), but it would be great to eventually land somewhere that doesn't require everyone to remember to configure this themselves.

Reply:
@bernerdschaefer I've been investigating our ELB (Cedar) values and they use the AWS default of 60s, defaulting to them would be a good start or the Dogwood values? I'm not aware of any best practices around this, do you have any suggestions?

Reply:
<img width="615" alt="Screen Shot 2020-03-17 at 10 30 20 AM" src="https://user-images.githubusercontent.com/193/76884060-54e66a80-683a-11ea-8d33-76f1cbc95143.png">

Looks like Dogwood also uses the default. I'm 👍 on defaulting the read/write timeouts to 60s to start with. If you don't know, you can do that by updating the struct tag to on the fields to be `env:"HTTP_SERVER_WRITE_TIMEOUT,default=60s"`.

Reply:
@bernerdschaefer ack and thanks 👍